Project_27/Project_27.cbp

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<CodeBlocks_project_file>
+	<FileVersion major="1" minor="6" />
+	<Project>
```

Comment:
please delete and do not commit oher except .c/cpp, *.h and Makefile files

Reply:
I thought that, this is my Makefile file (project file in Codeblocks)